cl_dll/hud_customtimer.cpp

```diff
@@ -34,7 +34,7 @@ int CHudCustomTimer::Draw(float time)
 	UnpackRGB(r, g, b, gHUD.m_iDefaultHUDColor);
 
 	FillRGBA((ScreenWidth / 2) - 50, (gHUD.m_scrinfo.iCharHeight * 4) - 6, (diff / m_flSeconds) * 100, 4, r, g, b, 210);
-	snprintf(str, sizeof(str), "Timer: %d", (int)diff);
+	snprintf(str, sizeof(str), "Timer: %.1f", diff);
```

Comment:
```suggestion
	snprintf(str, sizeof(str), "Timer: %.01f", diff);
```
I think this is needed for it to show .0

Reply:
It does show even without that change.